lib/ddr/models/solr_document.rb

```diff
@@ -113,7 +115,7 @@ def admin_policy
     end
 
     def has_children?
-      ActiveFedora::SolrService.class_from_solr_document(self).reflect_on_association(:children).present?
+      children.present?
```

Comment:
Is this change germane to the essential change in the PR? Not sure I object, but please explain.